src/ResourceManager/StorageSync/Commands.StorageSync/debug.ps1

```diff
@@ -0,0 +1,211 @@
+param(
```

Comment:
@afedyashov what is the purpose of this script?

Reply:
Debugging script loaded by F5, as per developer guide.

Reply:
Same comment above -- debugging is strictly for testing, we don't need to check-in anything relating to this.